pkg/virt-launcher/virtwrap/launchsecurity/virsh.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021
+ *
+ */
+
+package launchsecurity
+
+//go:generate mockgen -source $GOFILE -package=$GOPACKAGE -destination=generated_mock_$GOFILE
+
+/*
+ ATTENTION: Rerun code generators when interface signatures are modified.
+*/
+
+import (
+	"os/exec"
+)
+
+type Virsh interface {
+	Domcapabilities() ([]byte, error)
+}
+
+type virsh struct {
+}
+
+func NewVirsh() Virsh {
+	return &virsh{}
+}
+
+func (v *virsh) Domcapabilities() ([]byte, error) {
+	return exec.Command("virsh",
+		"domcapabilities",
+		"--machine", "q35",
+		"--arch", "x86_64",
+		"--virttype", "kvm",
+	).Output()
+}
```

Comment:
I think this is not needed as we can already obtain the capabilities in the virt-launcher.

Reply:
Do you mean it is better to pass the capabilities via `VirtualMachineOptions` from `virt-handler`? Initially I was looking at that option. But in practice that would require modifications of the node-labeler (since currently it does not save the complete  domcapabilities AFAIK) and some additional logic to send and recieve that data during VMI sync.... just calling `virsh domcapabilities` from `virt-launcher` turned out to be simpler and I think it should not cause issues.

Potentially at some point it would be usefull to also label a node as SEV-capable. Though at the moment I have not included that part here (it can be introduced in a follow-up PR) hence decided to go with the simple approach at first. Do you think it will be more beneficial to send the data from `virt-handler` at current step?

Reply:
UPD: I think my prev comment is outdated. After looking at it one more time I've updated the code and now the SEV configuration is obtained by node-labeller and virt-launcher gets it from virt-hanldler.

Reply:
One more update: I just realized that KubeVirt has recently switched to a newer libvirt (7.6.0). That makes the whole story much easier since we can leverage libvirt to automatically fill `cbitpos` and `reducedphysbits`. So no need to query domcapabilities anymore.